server/lib/src/idm/server.rs

```diff
@@ -677,6 +693,55 @@ pub trait IdmServerTransaction<'a> {
         })
     }
 
+    fn process_uuid_to_identity(
+        &mut self,
+        uuid: &Uuid,
+        ct: Duration,
+        source: Source,
+    ) -> Result<Identity, OperationError> {
+        let anon_entry = self
```

Comment:
To avoid the former confusion, lets comment here that users via ldap bind are *always* only granted anonymous equivalent rights unless they auth with an api-token. (Due to ldap binds being weaker than our normal auth flow)

Reply:
Agree, added.